src/popup/components/FungibleTokens/TokensListItem.vue

```diff
@@ -10,7 +11,7 @@
     }"
     :extend="preventNavigation"
     :selected="selected"
-    v-on="$listeners"
+    @click="$emit('click', $event)"
```

Comment:
This can be removed as the listeners are now part of the `$attrs` which are bind to this component.